www/include/monitoring/downtime/template/AddDowntime.ihtml

```diff
@@ -151,12 +151,17 @@
 {literal}
 
 <script type="text/javascript">
-
-    initDatepicker("datepicker", "mm/dd/yy", "0");
-
-    $(document).ready(function() {
-        turnOnEvents();
+    jQuery(".timepicker").each(function () {
+        if (!$(this).val()) {
+            $(this).val(moment().tz(localStorage.getItem('realTimezone')
+                ? localStorage.getItem('realTimezone')
+                : moment.tz.guess()).format("HH:mm")
+            );
+        }
     });
-
+    jQuery(".timepicker").timepicker();
```

Comment:
I don't quite understand what's intended... why make the ".timepicker" selection twice?

Reply:
first timepicked is meant to add values to each timepicker and timezone parameter.
second one is to instantiate them